h2o-r/h2o-package/R/frame.R

```diff
@@ -1182,11 +1182,14 @@ NULL
   if( !missing(col) ) {     # Have a column selector?
     if( is.logical(col) ) { # Columns by boolean choice
       col <- which(col)     # Pick out all the TRUE columns by index
-    } else if( base::is.character(col) ) {   # Columns by name
-      idx <- match(col,colnames(data)) # Match on name
-      if( any(is.na(idx)) ) stop(paste0("No column '",col,"' found in ",paste(colnames(data),collapse=",")))
-      col <- idx
-    }
+    } else if (base::is.character(col)) {
+       idx <- match(col, colnames(data))
+        if (any(is.na(idx))){
+          stop(paste0("No column '", col[which(is.na(idx))], "' found in ",
```

Comment:
Jira ticket just mention single column case. If you type two two columns that doesn't exists you will get
```
h=as.h2o(iris)
h[, c("a1","a2")]
#Error in `[.H2OFrame`(h, , c("a1", "a2")) : 
#  No column 'a1' found in Sepal.Length,Sepal.Width,Petal.Length,Petal.Width,SpeciesNo column 'a2' found in Sepal.Length,Sepal.Width,Petal.Length,Petal.Width,Species
```
Use `paste(col[is.na(idx)], collapse=",")` instead of `col[which(...)]` so multiple missing columns will be collapsed also.